paket.dependencies

```diff
@@ -1,12 +1,16 @@
+
+source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
+source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7-transport/nuget/v3/index.json
```

Comment:
Sources for hidden Microsoft.Build dlls that preview bits of F# 7 depend on. We probably gotta wait for these previews to be available.